cmake/sca/eclair/sca.cmake

```diff
@@ -0,0 +1,141 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2023, BUGSENG Srl
+
+find_program(ECLAIR_ENV eclair_env REQUIRED)
+message(STATUS "Found eclair_env: ${ECLAIR_ENV}")
+
+find_program(ECLAIR_REPORT eclair_report REQUIRED)
+message(STATUS "Found eclair_report: ${ECLAIR_REPORT}")
+
+if(ECLAIR_OPTIONS_FILE)
```

Comment:
Not blocking as we can continue work on Eclair integration, but should the initial value of `ECLAIR_OPTIONS_FILE` be fetched using `zephyr_get(ECLAIR_OPTIONS_FILE)` so that it also uses the intended file when invoked through sysbuild ?